src/flows/context/flow.query.tsx

```diff
@@ -129,6 +134,8 @@ export const FlowQueryProvider: FC = ({children}) => {
             timeRangeStart,
             timeRangeStop,
           },
+          // future use: additional Variables injected into notebooks
+          variables: flow?.range ? getTimeRangeVars(flow.range) : [],
```

Comment:
how is this different than the variables immediately above this?

Reply:
The focus is on using the typed objects (which we already have in TimeRange), and transforming with the typed helpers (which we already have), in order to produced the typed output (Variables) which is an abstraction used in all other helpers we need downstream.

Reply:
if that type you're looking for is `Variables`, you're going in the wrong direction. after many months of detangling broken dashboard lifecycles and hand crafting rendering rules to prevent the page from crashing if it included a variable, it was an intentional move to shift away from that implementation that conflates resource fetching / storage / selection / context of selection and all of that into a separate type that represents a variable selection and not a variable resource.

Reply:
The rest of the UI application (outside of notebooks) still uses the `Variable` abstraction. Also consulted with other senior devs who are still using this abstraction. For now, going to scope these changes to the rest of the UI application outside of notebooks.